Log Utility for yadl

karthik-u-s opened the pull request:
/_Simple log utility. Yet to be implemented in Yadl._/
Signed-off-by: Karthik U S kus.karthikus9@gmail.com

karthik-u-s committed:
Log Utility for yadl
/*Simple log utility. Yet to be implemented in Yadl.*/
Signed-off-by: Karthik U S <kus.karthikus9@gmail.com>

josephaug26 commented:
1) The utility is not thread safe
2) more info in commit message

josephaug26 commented on util/log/src/log_yadl.c:
```diff
@@ -0,0 +1,71 @@
+                vfprintf(fp,frmt,args);
+                va_end(args);
+        }
+}
```
Needs a new line after }

josephaug26 commented on util/log/src/log_yadl.h:
```diff
@@ -0,0 +1,49 @@
+ *       void
+ */
+void
+log_fin();
```
Needs a new line after ;

josephaug26 commented on util/log/src/Makefile:
```diff
@@ -0,0 +1,7 @@
+log:log_yadl.o liblog.a
```
1) The Makefile shouldnot be created by you. You need to create Makefile.am instead, which autogen.sh picks up and creates the Makefile for you.

 2) You need an entry in configure.ac for util/log/src/

http://www.delorie.com/gnu/docs/autoconf/autoconf_toc.html#SEC_Contents

read the chapters on autogen and configure

karthik-u-s committed:
Incermental patch of Log Utility
/*Integrated the log utility with yadl.
Have some problem in the automake.
Unit testing yet to be implemented*/

Signed-off-by: Karthik U S <kus.karthikus9@gmail.com>
